try:
    import poopy
    print('done')
except Exception as e:
    print(e)